crates/primitives-traits/Cargo.toml

```diff
@@ -16,7 +16,7 @@ workspace = true
 reth-codecs = { workspace = true, optional = true }
 
 # ethereum
-alloy-consensus = { workspace = true, features = ["k256"] }
+alloy-consensus = { workspace = true, features = ["k256", "crypto-backend"] }
```

Comment:
since we don't use this directly and users can always force this, perhaps we dont need this?